backend/app/services/project_service.py

```
@@ -0,0 +1,187 @@
+        # git rev-parse --git-dirコマンドを実行してGitリポジトリか確認
+        try:
+            process = await asyncio.create_subprocess_exec(
+                "git",
+                "rev-parse",
+                "--git-dir",
+                cwd=str(path_obj),
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.PIPE,
+            )
+            await process.communicate()
+            return process.returncode == 0
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)